pkg/icingadb/db.go

```diff
@@ -54,6 +53,10 @@ type Options struct {
 	// MaxRowsPerTransaction defines the maximum number of rows per transaction.
 	// The default is 2^13, which in our tests showed the best performance in terms of execution time and parallelism.
 	MaxRowsPerTransaction int `yaml:"max_rows_per_transaction" default:"8192"`
+
+	// WsrepSyncWait defines which kinds of SQL statements catch up all pending sync between nodes first, see:
+	// https://mariadb.com/kb/en/galera-cluster-system-variables/#wsrep_sync_wait
+	WsrepSyncWait int `yaml:"wsrep_sync_wait" default:"7"`
```

Comment:
It would be helpful if the comment mentioned Galera cluster outside of just the URL.

Also, this should also be documented properly. After all, wasn't the idea to have this configurable so that we use the safer option by default but in a setup where it isn't needed, this can be set to 0 if it's not necessary as connections are all routed to the same node?